apps/website/src/components/content/Section.tsx

```diff
@@ -1,40 +1,58 @@
-import React, { useMemo } from "react"
+import React, { useMemo } from "react";
 import topographyTexture from "../../assets/textures/topography.svg";
 import dustTexture from "../../assets/textures/dust.svg";
 
 type SectionProps = {
   children?: React.ReactNode;
   dark?: boolean;
+  offsetParent?: boolean;
   className?: string;
   containerClassName?: string;
 };
 
-const Section: React.FC<SectionProps> = ({ children, dark, className, containerClassName }) => {
+const Section: React.FC<SectionProps> = ({
+  children,
+  dark,
+  offsetParent = true,
+  className,
+  containerClassName,
+}) => {
   // Determine the classes for the two elements
-  const sectionClass = useMemo(() => [
-    'py-16 relative z-0',
-    dark && 'bg-alveus-green text-alveus-tan',
-    !dark && 'bg-alveus-tan text-alveus-green-900',
-    className,
-  ].filter(Boolean).join(' '), [ dark, className ]);
-  const containerClass = useMemo(() => [
-    'container mx-auto px-4',
-    containerClassName,
-  ].filter(Boolean).join(' '), [ containerClassName ]);
+  const sectionClass = useMemo(
+    () =>
+      [
+        "py-16",
+        offsetParent && "relative z-0",
+        dark && "bg-alveus-green text-alveus-tan",
+        !dark && "bg-alveus-tan text-alveus-green-900",
+        className,
+      ]
+        .filter(Boolean)
+        .join(" "),
+    [offsetParent, dark, className]
+  );
+  const containerClass = useMemo(
+    () =>
+      ["container mx-auto px-4", containerClassName].filter(Boolean).join(" "),
+    [containerClassName]
+  );
 
   // Determine the texture to use
-  const opacity = dark ? 'opacity-[0.06]' : 'opacity-[0.03]';
+  const opacity = dark ? "opacity-[0.06]" : "opacity-[0.03]";
   const texture = dark ? dustTexture.src : topographyTexture.src;
 
   return (
     <section className={sectionClass}>
-      <div
-        className={`absolute inset-0 -z-10 ${opacity}`}
-        style={{ backgroundImage: `url(${texture})`, backgroundSize: '32rem' }}
-      />
-      <div className={containerClass}>
-        {children}
-      </div>
+      {offsetParent && (
+        <div
+          className={`absolute inset-0 -z-10 ${opacity}`}
+          style={{
+            backgroundImage: `url(${texture})`,
+            backgroundSize: "32rem",
+          }}
+        />
+      )}
```

Comment:
Ah yes, thats left-over. I had the show-an-tell at one point request native fullscreen for the whole document and not only for the section. Then I needed to make the entries container `fixed inset-0` but that did not work because the section was promoted to a offset parent because of the background image. That should be solved now, that I request native fullscreen for the entries container which has basically the same effect as `fixed inset-0` had.

Reply:
Oh actually we still need this because you can get into the full screen mode without native fullscreen, e.g. if browsers do not support it or if gets closed by pressing ESC.

Reply:
Oh I see why this is needed now -- it makes more sense to me, but I worry that this is a fragile way to handle "fullscreen"? If any changes down the road happened to the layout etc. above this, it could easily end up not working as intended again?

To me, it feels like fullscreen would be better off being implemented in a way where it hoists itself far higher in the stack (basically right to the top of the page) -- maybe using a React portal?

Reply:
Yeah not super happy with that either, but I also did not really want to re-render all the content in the portal and handle the intersection observer twice etc. At that point I would just remove the "hot swapping" between views and make them separate completely and put them on different routes. I would wait and see if the feature even gets used. And if we change something in the layout that breaks this its not that bad to fix.